twine/__main__.py

```diff
@@ -25,7 +27,23 @@ def main() -> Any:
     try:
         return cli.dispatch(sys.argv[1:])
     except (exceptions.TwineException, requests.HTTPError) as exc:
-        return "{}: {}".format(exc.__class__.__name__, exc.args[0])
+        return _format_error(f"{exc.__class__.__name__}: {exc.args[0]}")
+
+
+def _format_error(err: str) -> str:
```

Comment:
Nitpick: more descriptive variable name:

```suggestion
def _format_error(message: str) -> str:
```

Reply:
Resolved